lib/restaurant.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'menu.rb'
+
+class Restaurant
+  attr_reader :menu, :order
+
+  def initialize
+    @menu = Menu.new
+    @order = {}
+  end
+
+  def make_order(dish, quantity)
+    fail "Cannot order items not on the menu" unless menu.on_menu?(dish)
+    if order.include?(dish)
+      order[dish] += quantity
+    else
+      order.store(dish, quantity)
+    end
+  end
+
+  def reciept
+    "Total cost: £#{total}\n Which is #{ind_total.join(", ")}"
+  end
+
+  def review
+    order.collect{|price, quantity| "order = #{dish}: #{quantity}"}.join(", ")
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `price`. If it's necessary, use `_` or `_price` as an argument name to indicate that it won't be used.<br>Style/CollectionMethods: Prefer `map` over `collect`.